core/code/boot.js

```diff
@@ -249,6 +249,16 @@ function prepPluginsToLoad() {
   };
 }
 
+/**
+ * update Z-Index of plugins buttons in leaflet container
+ */
+function updateControlBarZIndex() {
+  $('.leaflet-control-container .leaflet-top.leaflet-left').children().each((index, element) => {
+    element.style.zIndex = (10100 - index).toString();
```

Comment:
Can we move this into CSS?

Reply:
IDK, maybe there is a better solution.

it recalculates Z-Indexs to make sure first button is topmost and last button down.
This way subelements like the DrawTools "finsh"+"Cancel" are drawn above buttons in next columns.